python-sdk/src/astro/render/workflow_directory_parser.py

```diff
@@ -10,11 +10,51 @@
 from airflow.models import BaseOperator
 from yaml.scanner import ScannerError
 
+from astro.constants import GENERATED_WORKFLOW_INCLUDE_DIRECTORY
```

Comment:
uuffff yeah, I agree, I thought we would just have the ability to convert SQL files to task but this files involves a lot more than just that for example loading using YAML file -- that should just be in SQL CLI